src/states/common/approved.rs

```diff
@@ -168,7 +168,7 @@ pub trait Approved: Relocated {
                 }
             }
 
-            match self.chain_poll_all(outbox)? {
+            match self.chain_poll(outbox)? {
```

Comment:
Why that change here?

Reply:
This was a leftover from the state refactoring PRs. It should have been renamed back then, but it slipped through. As it is a trivial change, I though sneaking it into this PR is OK.